e2e/app/workspace-card.ts

```diff
@@ -41,24 +42,25 @@ export default class WorkspaceCard extends BaseElement {
   }
 
   static async findCard(page: Page, workspaceName: string): Promise<WorkspaceCard | null> {
-    const cards = await page.$x(WorkspaceCard.cardRootXpath);
-    for (const card of cards) {
-      const children = await card.$x(`.//*[@data-test-id="workspace-card-name" and text()="${workspaceName}"]`);
+    const allCards = await this.getAllCards(page);
```

Comment:
is this the workspace landing page? else can we name this as getAllWorkspaceCards